crates/api/src/reports/community_report/resolve.rs

```diff
@@ -25,6 +30,21 @@ pub async fn resolve_community_report(
 
   let community_report_view =
     CommunityReportView::read(&mut context.pool(), report_id, person_id).await?;
+  let site = Site::read_from_instance_id(
+    &mut context.pool(),
+    community_report_view.community.instance_id,
+  )
+  .await?;
+
+  ActivityChannel::submit_activity(
+    SendActivityData::SendResolveReportToSite {
+      object_id: community_report_view.community.ap_id.inner().clone(),
+      actor: local_user_view.person,
+      report_creator: community_report_view.creator.clone(),
+      site,
+    },
+    &context,
+  )?;
```

Comment:
I don't think it's a good idea to allow admins of another instance to resolve a local community report.

While reports of posts and comments are currently mostly considered to be mod responsibility, communities are always admin responsibility.
Communities can very much be violating the reporter's instance rules, such as a remote porn community on an instance not allowing porn, but the community instance might be fine with that.

IMO the only case when a community report should get resolved through federation is when the community gets removed by an admin of the community instance, in all other cases the report should stay around for local admin review.

Reply:
This should be opened as a separate issue

Reply:
it might make sense to extract the entire functionality of federating resolving of community reports if you consider that too much for this PR, but imo this shouldn't be merged with full federation of resolving those reports in the first place and then ripping it out again after merging this.

Reply:
What exact actions could a local admin take in response to a remote admin resolving a community report?

Reply:
best example is an instance that doesn't allow nsfw content, but it federates with instances that allow this.

a user on the non-nsfw instance reports the community for being nsfw, which the community instance admin resolves as not actionable, as it's allowed there. the admin of the reporter's instance will still want to see the report to remove the community locally.

Reply:
I fixed this problem by changing the filter that's used when listing unresolved reports.